Engine/Module/ModuleRender.cpp

```diff
@@ -514,3 +514,39 @@ GameObject* ModuleRender::GetRaycastIntertectedObject(const LineSegment & ray)
 	return selected;
 }
 
+bool ModuleRender::GetRayCastIntersectedPosition(const LineSegment & ray, float3& position)
```

Comment:
You can avoid this function using `Triangle::Intersects()` `intersectionPoint` parameter. For that you can use current Mouse piciking redifining the function `GetRaycastIntertectedObject` of `ModuleRender`.